src/telemetry/backend-metrics.yaml

```
@@ -0,0 +1,222 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+page:
+  view:
+    type: event
+    description: |
+      A web page view event distinguishable by the page’s URL.
```

```suggestion
    description: |
      A web page view event distinguishable by the page’s URL.
    send_in_pings:
      - events
```
We need to declare which ping this is going to be sent in, otherwise [glean_parser will complain](https://github.com/mozilla/glean_parser/blob/f484585a93793ace324adf08ef9da7890c3d763d/glean_parser/javascript_server.py#L161-L169). Also applies to events below.

---

Good catch!

@rhelmer I think `send_in_pings` is the one remaining thing that needs to be fixed before merging this to unblock https://github.com/mozilla/probe-scraper/pull/757 (incorporating `glean_parser` into the build process could be done in a separate PR).